task/htmlTask.js

```diff
@@ -0,0 +1,94 @@
+import {Task} from "./task.js";
+const DONE_TASK_TEXT_CLASS = "task-input-text-corssed";
+const DONE_TASK_BUTTON_TEXT = '\u{02713}';
+const DELETE_TASK_BUTTON_TEXT = "X";
+
+export class htmlTask extends Task{
+    constructor(onUpdate, onDelete, onDone, onUnDone){
+        super(onUpdate, onDelete, onDone, onUnDone);
+        this.div = document.createElement('div');
+        this.div.classList.add("task");
+        this.div.task = this;
+
+        this.inputBar = undefined;
```

Comment:
why?

Reply:
I don't want to create a new attribute in a separate function, all the definitions of attributes should be in the constructor, this way, by looking at the constructor you can see all the attributes that the class has.
Makes for easier to read and debug code.

Reply:
we solve this problem with typescript, not with long constructor functions